stix_shifter_modules/aws_guardduty/configuration/lang_en.json

```diff
@@ -0,0 +1,32 @@
+{
+    "connection": {
+        "region": {
+            "label": "AWS GuardDuty Region Name",
+            "description": "Specify the region name of the GuardDuty data source"
+        },
+        "help": {
+            "label": "Need additional help?",
+            "description": "More details on the data source setting can be found in the specified link"
+        },
+        "detector_ids": {
+            "label": "Detector Ids",
+            "description": "Detector id of the GuardDuty"
```

Comment:
Description needs to be more clear. According to this comment https://github.com/opencybersecurityalliance/stix-shifter/pull/1516#discussion_r1260065211

You may have to show an example of how to specify the ids.

Reply:
Updated the description of the detector id with an example.